smart_campus/app/templates/app/base.html

```diff
@@ -8,38 +8,65 @@
+
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/divider.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/segment.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/form.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/input.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/button.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/list.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/message.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/icon.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/dropdown.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "Semantic-UI/components/transition.css" %}">
+    <link rel="stylesheet" type="text/css" href="{% static "app/css/base.css" %}">
   {% endblock %}
 </head>
 
 <body>
   {% block nav %}
-  <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
-    <a class="navbar-brand" href="#">管理後台</a>
-    <button class="navbar-toggler d-lg-none" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-
-    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
-      <ul class="navbar-nav mr-auto">
-        <li class="nav-item active">
-          <a class="nav-link" href="{% url 'index' %}">Home <span class="sr-only">(current)</span></a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="{% url 'Station List Page' %}">站點管理</a>
-        </li>        
-      </ul>
-      <ul class="navbar-nav right">
-        {% if email %}
-          <li class="nav-item ">
-            <a class="nav-link disabled" >{{ email }}</a>
-          </li>
-        {% endif %}
-        <li class="nav-item ">
-          <a class="nav-link" href="{% url 'Logout Page' %}">登出</a>
-        </li>
-      </ul>
+  <!-- <div class="ui main container"> -->
+    <div class="ui borderless main menu" id="selection">
+      <a class="header item" href="{% url 'index' %}">成大智慧校園管理平台</a>
+      <div class="ui dropdown item" id="selection">
+        管理站點及通知
+        <i class="dropdown icon"></i>
+        <div class="menu">
+          {% if categories %}
+            {% for category in categories %}
+              <div class="item" href="">{{ category.name }}</div>
+            {% endfor %}
+          {% endif %}
+          <a class="item" href="{% url 'Station List Page' %}">全部站點</a>
+          <hr>
+          <a class="item" href="{% url 'Category Add Page' %}">新增類別</a>
+          <!-- <a class="item" href="">新增站點</a> -->
```

Comment:
**DO NOT** version control **ANY** code that is commented out.

Reply:
okay, learned it.